pkgs/applications/science/machine-learning/vowpal-wabbit/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, stdenv, fetchFromGitHub, cmake, boost169, rapidjson, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "vowpal-wabbit";
+  version = "8.9.2";
+
+  src =
+    fetchFromGitHub {
+      owner = "VowpalWabbit";
+      repo = "vowpal_wabbit";
+      rev = version;
+      sha256 = "0np11nvxvbxgfdlm4q07lw7zni6mm4l93bf7pj0rif7fz15k8f3l";
+    };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [
+    boost169
+    rapidjson
+    zlib
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DVW_INSTALL=ON"
+    "-DBUILD_TESTS=OFF"
```

Comment:
A comment explaining why tests are disabled would be nice.

Reply:
The default cmake parameters also turn off tests `-DBUILD_TESTING=OFF`.